cmd/harborAPI.go

```diff
@@ -129,6 +130,27 @@ func update(token string, url string, data interface{}) {
 	}
 }
 
+// GetLogs returns all logs for a shipment
+// TODO: add a tail flag feature (jkurz)
```

Comment:
good point. I'll play around with docker-compose logs more and implement features more like they do.